source/index.html.erb

```diff
@@ -1,5 +1,27 @@
----
-title: Jeff Stolz | Product Designer
----
-
-
+<main class="index">
+    <h1><%= t("index.hello") %></h1>
+    <h3><%= t("index.welcome") %></h3>
+    <section class="index__cards">
+        <% link_to 'mobile-heartbeat.html', :class => "index__case" do %>
+            <div class="index__case--overlay">
+                <p class="index__case-description"><%= t("index.mh-description") %></p>
+            </div>
+            <h4 class="index__case-heading"><%= t("index.mh") %></h4>
+            <%= image_tag 'mh-thumb.png', :class => "index__thumbnail", :alt => t("index.mh-alt") %>
+        <% end %>
+        <% link_to 'hmh-marketplace.html', :class => "index__case" do %>
```

Comment:
I notice throughout these files there's a mix of double and single quotes. We like to stick to double quotes (https://github.com/thoughtbot/guides/tree/master/style/sass), so we can reserve single quotes for things like concatenating strings.